jedi_vim.py

```diff
@@ -254,7 +254,7 @@ def goto(mode="goto", no_output=False):
                         tagname = d.name
                         while vim_eval('taglist("^%s$")' % tagname) != []:
                             tagname = d.name + ' ' + ''.join(
-                                [random.choice(string.lowercase)
+                                [random.choice(string.lower)
```

Comment:
Why are we even joining the same thing four times? Besides: string.lower is a function and I don't even find a definition for `string.`